modules/administration-guide/pages/about-persistent-user-home.adoc

```diff
@@ -58,5 +62,4 @@ Due to requirement three, the default UDI image for example adds the `/home/user
 RUN stow -t /home/user/ -d /home/tooling/ --no-folding 
 ----
 
-in the Dockerfile so that `/home/tooling` is accessible from `/home/user` even when not using the `persistUserHome` feature.
-
+in the Dockerfile so that files in `/home/tooling` is accessible from `/home/user` even when not using the `persistUserHome` feature.
```

Comment:
Sorry something I missed, "is" should be changed to "are" now:
"...so that files in `/home/tooling` *are* accessible from..."

Reply:
No problem, fixed